### DISCUSSION TITLE
Display customFields of Workitems

### DISCUSSION THREAD

**Discussion by hafnerpw** [rfennell/AzurePipelines#1187] [Q&A]
Hi, in my Azure I have custom Fields that I would like to display in the release notes. In my case its Story Points inside "Planning". But in general is there any way to get this information in the handlebars template. I tried checking all object with {{json}} but could not find anything.

**Comment by rfennell** [comment]
Yes you can. The basic form is

```
{{#forEach workItems}}
*  **{{this.id}}**  {{lookup this.fields 'System.Title'}}
   - **WIT** {{lookup this.fields 'System.WorkItemType'}} 
   - **Tags** {{lookup this.fields 'System.Tags'}}
   - **Custom Field** {{lookup this.fields 'CustomFieldName'}}
   - **Assigned** {{#with (lookup this.fields 'System.AssignedTo')}} {{displayName}} {{/with}}
{{/forEach}}
```

To find the custom field name you can use the `json` dump to see the whole object

```
{{#forEach workItems}}
{{ json this}}
{{/forEach}}
```

**Reply by hafnerpw** [reply]
Hey there, I am so sorry! It was working all the Time. But when the Field is empty then the key won't exists. I expected it to have empty value But thanks for your fast response, you're the best👍
